packages/graphql/src/contracts.js

```diff
@@ -182,6 +182,8 @@ export function setNetwork(net, customConfig) {
 
   setMarketplace(config.V00_Marketplace, config.V00_Marketplace_Epoch)
   setIdentityEvents(config.IdentityEvents, config.IdentityEvents_Epoch)
+  setProxyFactory(config.ProxyFactory, config.ProxyFactory_Epoch)
+  // TBD: do we need those 2 lines ?
```

Comment:
it's used in utils/proxy.js but we could refer the context.proxyFactory there instead now we're setting it here

Reply:
Cleaned up. Sorry I had no pushed my latest changes.